### ISSUE TITLE
Request for support oh-my-zsh

### ISSUE THREAD

**Issue by CarlCao17** [microsoft/inshellisense#19]
**Is your feature request related to a problem? Please describe.**
I'm using oh-my-zsh on zsh and try to execute `is -s zsh`. But the terminal will changed to the zsh shell, not my oh-my-zsh shell appearance. I want to use my oh-my-zsh settings while enjoy inshellisense services.

**Describe the solution you'd like**
None

**Describe alternatives you've considered**
None

**Additional context**

**Comment by CarlCao17** [comment]
And I found that when I open terminal in Jetbrains IDE like Golang, there are some output or hints:

```shell

[WARNING]: Console output during zsh initialization detected.

When using Powerlevel10k with instant prompt, console output during zsh
initialization may indicate issues.

You can:

  - Recommended: Change ~/.zshrc so that it does not perform console I/O
    after the instant prompt preamble. See the link below for details.

    * You will not see this error message again.
    * Zsh will start quickly and prompt will update smoothly.

  - Suppress this warning either by running p10k configure or by manually
    defining the following parameter:

      typeset -g POWERLEVEL9K_INSTANT_PROMPT=quiet

    * You will not see this error message again.
    * Zsh will start quickly but prompt will jump down after initialization.

  - Disable instant prompt either by running p10k configure or by manually
    defining the following parameter:

      typeset -g POWERLEVEL9K_INSTANT_PROMPT=off

    * You will not see this error message again.
    * Zsh will start slowly.

  - Do nothing.

    * You will see this error message every time you start zsh.
    * Zsh will start quickly but prompt will jump down after initialization.

For details, see:
https://github.com/romkatv/powerlevel10k/blob/master/README.md#instant-prompt

-- console output produced during zsh initialization follows --

/Users/bytedance/.inshellisense/key-bindings.zsh:10: parse error near `__inshellisense'

```

**Comment by Programie** [comment]
In my case I'm using zsh with [powerlevel10k](https://github.com/romkatv/powerlevel10k) and can confirm the same issue. The shell style is completely changed by inshellisense.

This is caused, because the `$PS*` environment variables (i.e. `$PS1`, `$PS2`, etc.) are unset by inshellisense.

I would expect inshellisense to extend the shell but not replace the whole style of it.

**Comment by cpendery** [comment]
Closing as this was fixed in [0.0.1-rc.5](https://github.com/microsoft/inshellisense/releases/tag/0.0.1-rc.5)
